README.md

```diff
@@ -2209,6 +2211,7 @@ The following table describes the most important command line arguments for `ai8
 | `--no-unload`            | Do not create the `cnn_unload()` function                    |                                 |
 | `--no-kat`               | Do not generate the `check_output()` function (disable known-answer test)  |                   |
 | `--no-deduplicate-weights` | Do not deduplicate weights and and bias values             |                                 |
+| `--scale-output` | Use scales from the checkpoint to recover output range while generating `cnn_unload()` function | |
 
```

Comment:
Why have we added this as an option? Shouldn't it be the default behavior, Is there any case where we do not want this?

Reply:
I agree, also in synthesis side, we have set this one's default value as False, I was going to write a comment on that too as I thought a True default would be better. If you are going to update the option that would handle that as well. just fyi